src/Java.Interop.Localization/Resources.resx

```diff
@@ -259,8 +270,9 @@ Values in quotes should not be translated.</comment>
 {1} - .NET member.</comment>
   </data>
   <data name="Generator_BG8A00" xml:space="preserve">
-    <value>Metadata element '{0}' matched no nodes.</value>
-    <comment>{0} - XML transform. (ex: '&lt;remove-node path="/api/package[@name='javax.sql']"')</comment>
+    <value>Metadata.xml element '{0}' matched no nodes.</value>
+    <comment>{0} - XML transform. (ex: '&lt;remove-node path="/api/package[@name='javax.sql']"')
```

Comment:
No worries if it's how you'd prefer to keep it, but maybe to avoid the abbreviation:

```
<comment>{0} - XML transform. (example: '&lt;remove-node path="/api/package[@name='javax.sql']"')
```